README.md

```diff
@@ -55,7 +55,7 @@ Skrop provides a set of filters, which you can use within the routes:
 * **crop(width, height, type)** — crops the image to have the specified width and height the type can be "north", "south", "east" and "west"
 * **cropByHeight(height, type)** — crops the image to have the specified height
 * **cropByWidth(width, type)** — crops the image to have the specified width
-* **resize(width, height, opt-keep-aspect-ratio)** — resizes an image. Third parameter is optional: true to keep aspect ratio, false to ignore (default to true)
+* **resize(width, height, opt-keep-aspect-ratio)** — resizes an image. Third parameter is optional: "ignoreAspectRatio" to ignore the aspect ratio, anything else to keep it
```

Comment:
nevermind. 
from the example I assumed you need to pass the third parameter, which is only true when it matches the string.
so we can use
```
resize(width, height)
resize(width, height, "keepAspectRatio")
resize(width, height, "something")
```